test/cypress/e2e/send.spec.js

```diff
@@ -162,29 +162,44 @@ describe('Send', () => {
   });
 
   /**
-   * Initialization dialogue and functionality is shown for the fresh account
+   * Initialization dialogue and functionality shouldn't be shown through wallet send.
+   * @expect initialization dialogue is shown
    * @expect successfully go through initialization
    * @expect transfer transaction appear with correct data
    * @expect initialization dialogue is not shown anymore
    */
-  it('Should be able to init account when needed', () => {
+  it('Should show initialize banner with account not initialized', () => {
     cy.autologin(accounts['without initialization'].passphrase, networks.devnet.node);
     cy.reload();
-    cy.visit(urls.send);
+    cy.visit(urls.dashboard);
+    cy.get(ss.initializeBanner).should('exist');
+    cy.get(ss.initializeBanner).find('a').click();
+    cy.get(ss.backButton).click();
     cy.get(ss.accountInitializationMsg).get(ss.accountInitializationBtn).click();
     cy.get(ss.sendBtn).click();
     cy.get(ss.resultMessage).should('have.text', msg.transferTxSuccess);
     cy.get(ss.accountInitializationMsg).should('not.exist');
     cy.wait(txConfirmationTimeout);
     cy.reload();
-    cy.visit(urls.send);
-    cy.get(ss.accountInitializationMsg).should('not.exist');
     cy.visit(urls.wallet);
     cy.get(ss.transactionRow).eq(0).as('tx');
     cy.get('@tx').find(ss.transactionAddress).should('have.text', accounts['without initialization'].address);
     cy.get('@tx').find(ss.transactionReference).should('have.text', 'Account initialization');
```

Comment:
Its worthy to check that the initialization banner is not shown on the dashboard anymore